src/main/java/tutoraid/model/StudentBook.java

```diff
@@ -85,13 +85,6 @@ public void setStudent(Student target, Student editedStudent) {
         students.setStudent(target, editedStudent);
     }
 
-    /**
-     * Refreshes the lesson book.
-     */
-    public void refreshStudentBook() {
```

Comment:
Would like to clarify the removal of this, and other refresh methods

Reply:
It's a method that I naively added to help fix the link. I removed it now since it is not working.